src/LoggerPlugin.php

```diff
@@ -15,31 +16,23 @@
  *
  * @author Joel Wurtz <joel.wurtz@gmail.com>
  */
-final class LoggerPlugin implements Plugin
+final readonly class LoggerPlugin implements Plugin
 {
-    use VersionBridgePlugin;
-
-    private $logger;
-
-    private $formatter;
-
-    public function __construct(LoggerInterface $logger, Formatter $formatter = null)
-    {
-        $this->logger = $logger;
-        $this->formatter = $formatter ?: new SimpleFormatter();
+    public function __construct(
+        private LoggerInterface $logger,
+        private Formatter $formatter = new SimpleFormatter()
```

Comment:
i think this is a small BC break: before, you could call `new LoggerPlugin($logger, null)`, now you must not pass a second parameter anymore.

Reply:
Good one, fixed it!